common/chain.go

```diff
@@ -158,3 +165,15 @@ func GetChainFromChainID(chainID int64) *Chain {
 	}
 	return nil
 }
+
+func GetBTCChainParams(chainID int64) (*chaincfg.Params, error) {
+	switch chainID {
+	case 18444:
+		return &chaincfg.RegressionNetParams, nil
+	case 18332:
+		return &chaincfg.TestNet3Params, nil
+	case 8332:
+		return &chaincfg.MainNetParams, nil
+	}
+	return nil, fmt.Errorf("chain (%d) is not a supported Bitcoin chain", chainID)
```

Comment:
```suggestion
func GetBTCChainParams(chainID int64) (p chaincfg.Params, err error) {
	switch chainID {
	case 18444:
		return chaincfg.RegressionNetParams, nil
	case 18332:
		return chaincfg.TestNet3Params, nil
	case 8332:
		return chaincfg.MainNetParams, nil
	}
	return p, fmt.Errorf("chain (%d) is not a supported Bitcoin chain", chainID)
```
Is there a reason to return a pointer? Returning the address of the config variable from `btcd` doesn't look clean IMO, we could return a value and use 
```
btcutil.NewAddressWitnessPubKeyHash(witnessProgram, &chainParams)
```
WDYT?

Reply:
Anything against this? I really think we should use `&chaincfg.RegressionNetParams` reference, etc.